### PR TITLE
Add/update GitHub namespace extensions

### PR DESCRIPTION
**Pull request by sau226**
One of the GitHub extensions (vscode pull request) has received an extension update.

The other was removed from the publish file for some reason and has not been updated - a user had filed an issue about this in the extension's repo (primer/github-vscode-theme#171) and the maintainer has previously filed a PR to publish this extension (PR #63)

### CODE REVIEW

**Comment by jankeromnes** [comment]
Hi @sau226, many thanks for fixing this!

> One of the GitHub extensions (vscode pull request) has received an extension update.

Great catch! 🎯

> The other was removed from the publish file for some reason and has not been updated - a user had filed an issue about this in the extension's repo (primer/github-vscode-theme#171) and the maintainer has previously filed a PR to publish this extension (PR #63)

The extension `GitHub.github-vscode-theme` was removed from this repository's auto-publish list in https://github.com/open-vsx/publish-extensions/commit/aea1532d7a4eb8a31ec8fb1949fd0684897c74fd because the lead developer on the https://github.com/github/vscode-codeql/ project (@aeisenberg) claimed the OpenVSX namespace `GitHub` in https://github.com/EclipseFdn/open-vsx.org/issues/267.

However, since no updates were published for https://open-vsx.org/extension/GitHub/github-vscode-theme since January, I think we can resume auto-publishing `GitHub.github-vscode-theme` from here for now. 👍

File: extensions.json
Comment (jankeromnes) [inline extensions.json:582]: It would probably be safer to checkout and pin a specific release, rather than to publish every day from the default branch:

```suggestion
      "repository": "https://github.com/primer/github-vscode-theme",
      "version": "4.1.1",
      "checkout": "v4.1.1",
```

**Review by jankeromnes — APPROVED** [review]
Thanks again!

Approved with one suggestion (which I can apply in one click before merging 🙌).

**Comment by jankeromnes** [comment]
> Approved with one suggestion (which I can apply in one click before merging 🙌).

Ah, I cannot. Please see my suggestion, apply it if it makes sense to you, and I'll be happy to squash & merge this Pull Request.

**Comment by aeisenberg** [comment]
Hi, just trying to understand what happened here. Looks like I was given control of the entire namespace, but unfortunately, I only publish the vscode-codeql extension. Is it possible to ensure that all other github extensions are auto-published?

**Comment by jankeromnes** [comment]
Hi @aeisenberg! Many thanks for publishing the vscode-codeql extension, and for checking in here. 🙏

Also, no worries -- I think OpenVSX's namespace-claiming process and auto-publishing policy should be slightly adjusted to better handle namespaces that are used for multiple extensions / maintainers / organizations (e.g. the first person to claim or join a namespace probably shouldn't bear the responsibility of publishing all the namespace's extensions).

Meanwhile, with this Pull Request, we'll resume the automated publishing of `GitHub.github-vscode-theme` and `GitHub.vscode-pull-request-github` which were disabled when the namespace was first claimed, so no harm done.

Also, FYI, OpenVSX namespace owners can already invite co-owners to their namespace, e.g. to facilitate the publishing of multiple extensions. The OpenVSX bot used in this repository has special rights to publish to any namespace (but with a visible warning on the extension's page), in order to auto-publish any extensions where the owners/maintainers couldn't set that up yet.

**Comment by brianking** [comment]
> Also, no worries -- I think OpenVSX's namespace-claiming process and auto-publishing policy should be slightly adjusted to better handle namespaces that are used for multiple extensions / maintainers / organizations (e.g. the first person to claim or join a namespace probably shouldn't bear the responsibility of publishing all the namespace's extensions).

Note that the namespace owner can add contributors to the namespace, who can then publish extensions (once they have signed the publisher agreement as well).

**Comment by jankeromnes** [comment]
Ping @sau226 about https://github.com/open-vsx/publish-extensions/pull/341#issuecomment-839587565 (please click on "apply" whenever you have some time)

**Comment by sau226** [comment]
@jankeromnes Thanks for the review - I've gone ahead and applied your suggestion.

**Comment by jankeromnes** [comment]
Perfect, many thanks! 🚀

**Pull request merged**
